resources/cloud-config/admiral.yaml

```diff
@@ -1,8 +1,8 @@
 #cloud-config
 
-coreos: 
+coreos:
   etcd2:
-    proxy: on
+    discovery: https://discovery.etcd.io/59b91eba6a0705e55ff470943ee5c14a
```

Comment:
@hazim1093 you confirmed we can use the same? or we shld create a ticket that we need to handle to create it dynamically and unique each time?